### PR TITLE
Add heatmap and discussion

### PR DESCRIPTION
**Pull request by trangdata**

### CODE REVIEW

File: content/20.results.md [hunk 1]
```diff
@@ -67,10 +67,12 @@ Our Wiki2019-LSTM returns, for each given name, a probability of that name origi
 We observed a multiclass area under the receiver operating characteristic curve (AUC) score of 95.4% for the classifier, indicating that the classifier can recapitulate name origins with high sensitivity and specificity.
 For each individual region, the high AUC (above 94%, Fig. {@fig:wiki2019_lstm}A) suggests that our classifier was sufficient for use in a broad-scale examination of disparities.
 We also observed that the model was well calibrated (Fig. {@fig:wiki2019_lstm}B).
+The confusion heatmap of log fold change of predicted over expected probability shows high multi-region classification accuracy compared to random chance (Fig. {@fig:wiki2019_lstm}C).
```
Comment (cgreene) [inline content/20.results.md]:
```suggestion
We also examined potential systematic errors between pairs of nationality groupings with a confusion heatmap and did not find off-diagonal enrichment for any pairing (Fig. {@fig:wiki2019_lstm}C).
```

File: content/20.results.md [hunk 2]
```diff
@@ -67,10 +67,12 @@ Our Wiki2019-LSTM returns, for each given name, a probability of that name origi
...
 
 ![The Wiki2019-LSTM model ranks the true nationality of Wikipedia names highly on testing data.
 The area under the ROC curve is above 94% for each category, showing strong performance regardless of nationality (A).
-Calibration curve shows consistency between the predicted probabilities (midpoints of each fixed-width bin) and the observed fraction of names in each bin (B).
+Calibration curve, computed with the caret R package, shows consistency between the predicted probabilities (midpoints of each fixed-width bin) and the observed fraction of names in each bin (B).
```
Comment (cgreene) [inline content/20.results.md]:
```suggestion
A calibration curve, computed with the caret R package, shows consistency between the predicted probabilities (midpoints of each fixed-width bin) and the observed fraction of names in each bin (B).
```

File: content/20.results.md [hunk 3]
```diff
@@ -67,10 +67,12 @@ Our Wiki2019-LSTM returns, for each given name, a probability of that name origi
...
+Log fold change probability confusion heatmap shows high multi-region classification accuracy compared to chance (C). Mean probability value in each predicted region is divided by the number of true names in that region (scaled to expected accuracy) to account for the imbalance of regions in the testing dataset.
```
Comment (cgreene) [inline content/20.results.md]:
```suggestion
A log fold change probability confusion heatmap shows high multi-region classification accuracy compared to chance (C). The mean probability value in each predicted region is divided by the number of true names in that region (scaled to expected accuracy) to account for the imbalance of regions in the testing dataset.
```

**Review by cgreene — APPROVED** [review]
Some thoughts but changes look good to merge to me! 👍

File: content/20.results.md (hunk 3 above)
Comment (dhimmel) [inline content/20.results.md]:
```suggestion
Heatmap showing whether names from a given region (x-axis) received higher (purple) or lower (green) predictions for each region (y-axis) than would be expected by region prevalence alone (C).
The values represent log~2~ fold change between the average predicted probability and the prevalence of the corresponding predicted region in the testing dataset (null).
Scaling by region prevalence accounts for the imbalance of regions in the testing dataset.
In all cases, the classifier predicts the true region above the expected null probability (matrix diagonal is all purple).
For off-diagonal cells, darker green indicates a lower mean prediction compared to the null.
For example, the classifier does not often mistake Hispanic names as Israeli, but is more prone to mistaking Muslim names as South Asian.
```

**Review by dhimmel — APPROVED** [review]

**Pull request merged**